### DISCUSSION TITLE
How to use and set up? - 設定方法などについての雑談・相談の場

### DISCUSSION THREAD

**Discussion by pafuhana1213** [pafuhana1213/KawaiiPhysics#66] [Q&A]
いつもKawaii Physicsプラグインを使ってくれてありがとうございます！
Thank you for always using the Kawaii Physics plugin!

ときどき「自分のキャラクタにプラグインを使ったけど上手く動作しない。どうすればいいの？」
という項目が**Issues**に登録されます。
しかし、Issueは不具合報告の場所ですし会話するにはあまり適していません。
Sometimes an issue is registered with 
the item "I used a plug-in for my character, but it doesn't work. What should I do?" 
However, Issues are a place to report bugs and are not very good for talking to each other.

そのため、プラグインの活用・設定方法・Tipsに関して共有し合う場所を試験的に用意してみました。
ここでユーザ間でコミュニケーションがあると嬉しいです。
For that reason, I tried to prepare a place to share information about plugin usage, setting methods, and tips.
It would be nice if there was communication between users here.

ちなみに、設定方法について相談する際は
他の人が詳細が分かるように画像や動画があるといいです。アセットが共有されると最高ですね！
By the way, when discussing the setting method, it would be nice to have images or videos 
so that other people can understand the details. Also, it's great when assets are shared!

ぜひ活用してください！
Please use it! Thanks!

**Comment by muchogrande** [comment]
Kawaii is すばらしい! Thanks so much for your hard work pafuhana1213!

For anyone getting started, I found this video very helpful for learning basics and best practices : https://www.youtube.com/watch?v=H4Gf6XMWh8I

**Comment by yukitoa** [comment]
お世話になっております。
KawaiiPhysicsの使用を検討しているのですが、ゲーム中(WorldSpace)での挙動とペルソナビュー（MeshSpace)での挙動が
アクタの移動の影響で大きく違って見えます。
出来ればペルソナビューでの見た目にある程度近づけたいのですが、アクタ移動の影響度を変更するパラメータなどは御座いますでしょうか？
以上よろしくお願いいたします。

**Comment by BineeMan** [comment]
@pafuhana1213 Greetings! I am using Kawaii physics for the skirt physics. I have a problem with setting up Kawaii physics. The issue is that when the character doesn't move, it looks like that Kawaii physics doesn't calculate joints position and the skirt returns to its initial position as it was rigged to the character which looks odd. When the character moves, it works fine. I've tried changing physics settings, but it didn't help. On the screenshot the character has Kawaii physics turned on and the skirt doesn't have any physics, the skirt doesn't cling to the leg.
Best wishes, BineeMan.
![Снимок экрана 2024-02-21 215528](https://github.com/pafuhana1213/KawaiiPhysics/assets/103749855/86e61bdf-0b46-46c1-b471-49fe5f13944a)
![Снимок экрана 2024-02-21 215045](https://github.com/pafuhana1213/KawaiiPhysics/assets/103749855/497154a5-1833-4e93-803f-09227c688421)
